README.md

```diff
@@ -131,6 +131,24 @@ Creates an Etch Packet and optionally sends it to the first signer.
       }
     ```
 
+##### getEtchPacket(options)
+
+Gets the details of an Etch Packet.
+* `options` (Object) - An object with the following structure:
+  * `variables` (Object) - Requires `eid`
+    * `eid` (String) - your Etch Packet eid
+  * `responseQuery` (String) - _optional_ A GraphQL Query compliant query to use for the data desired in the mutation response. Can be left out to use default.
+  * `query` (String) - _optional_ If you'd like complete control of the GraphQL query, you can pass in a GraphQL Query compliant string that will be used in the query call. This string should also include your response query, as the `responseQuery` param is ignored if `query` is passed. Example:
+    ```graphql
+      query GetEtchPacket (
+        $eid: String!,
+      ) {
+        etchPacket (
+          eid: $eid,
+        ) ${responseQuery}
```

Comment:
I think this is a little too cut-and-paste from javascript land...and you should replace the `${responseQuery}` with some actual psuedo fields. Something like what's done in [this area](https://www.npmjs.com/package/@anvilco/anvil#createetchpacketoptions).

Reply:
Good call. I went ahead and followed the format you described.